packages/server/lib/modes/run.ts

```diff
@@ -1069,8 +1059,16 @@ async function ready (options: { projectRoot: string, record: boolean, key: stri
     errors.throwErr('UNSUPPORTED_BROWSER_VERSION', browser.warning)
   }
 
+  let protocolManager: ProtocolManager | undefined
+
   if (browser.family === 'chromium') {
     chromePolicyCheck.run(onWarning)
+
```

Comment:
if protocol is enabled but the browser isnt chromium based, should we print a warning to the console?

Reply:
This section is actually what enables or disables it. We don't know until now whether or not it should be turned on or off with respect to the browser being used.

Reply:
But it has context of protocol being on and what browser we are in, right? So if I am running firefox and I have protocol on, we could log something like `protocol is  only compatible with chromium based browsers. It will not be enabled for this run` as like a `console.warn`? or does that happen someplace else?

Reply:
Oh like if we get it from the cloud that protocol is on? Yeah that'd be here:

https://github.com/cypress-io/cypress/blob/feat/protocol/packages/server/lib/cloud/api.ts#L383-L389

I'll sync with folks to see if that's something we want to do, but could likely be a separate PR.